command/test-fixtures/hcl2_upgrade_basic/expected.pkr.hcl

```diff
@@ -7,6 +7,15 @@
 # once they also need to be in the same folder. 'packer inspect folder/'
 # will describe to you what is in that folder.
 
+# Avoid mixing go templating calls ( ```for example '{{ upper `string` }}``` )
```

Comment:
```suggestion
# Avoid mixing go templating calls ( for example ```{{ upper `string` }}``` )
```

Reply:
oh yeah that looks right.

Reply:
still not seeing this fix in the current diff.